requirements.txt

```diff
@@ -1,3 +1,4 @@
+# List of packages which are needed for development and deployment
 numpy==1.14.0
 sklearn
 typing
```

Comment:
I think _typing_ is already included in the standard library

Reply:
But only for newer python versions, old ones need this extra package.